pages/premium.php

```diff
@@ -578,12 +578,36 @@ function premium() {
                 // Confirm change in seal... time will not be reimbursed
                 if(!isset($_POST['change_forbidden_seal'])) {
                     $confirmation_type = 'change_forbidden_seal';
-                    $confirmation_string = "Are you sure you would like to change from your {$player->forbidden_seal->name}?<br />
-                    You will lose {$system->time_remaining($player->forbidden_seal->seal_time_remaining)} of premium time.<br />
-                    <b>This can not be undone!</b>";
+                    if ($player->forbidden_seal->name == "Twin Sparrow Seal") {
+                        // Convert remaining premium time to days and calculate AK value
+                        $akCredit = floor(floor($player->forbidden_seal->seal_time_remaining / 86400) / 6);
```

Comment:
Since we have to do this calculation in more than once spot, making it a method would make maintenance easier.
`$akCredit = ForbiddenSeal::calcRemainingCredit();`